pkg/synccompactor/compactor_test.go

```
@@ -145,8 +183,9 @@
 
 	// Create compactor
 	compactableSyncs := []*CompactableSync{firstCompactableSync, secondCompactableSync}
-	compactor, err := NewCompactor(ctx, outputDir, compactableSyncs, WithTmpDir(tmpDir))
+	compactor, cleanup, err := createCompactor(compactableSyncs)
 	require.NoError(t, err)
+	defer cleanup()
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)